app/models/course/lecture.rb

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+class Course::Lecture < ActiveRecord::Base
+  acts_as_readable on: :updated_at
+  has_many_attachments on: :content
+
+  belongs_to :course, inverse_of: :lectures
+
+  scope :sorted_by_date, -> { order(start_at: :desc) }
+  scope :sorted_by_sticky, -> { order(sticky: :desc) }
```

Comment:
From the schema definition, there's no sticky attribute. I also think lectures don't need sticky.